core/arch/arm/include/kernel/delay.h

```diff
@@ -29,7 +30,22 @@
 #ifndef __KERNEL_DELAY_H
 #define __KERNEL_DELAY_H
 
+#include <stdbool.h>
+#include <stdint.h>
+
+#define US2CNT(m)	(((uint64_t)m * (uint64_t)read_cntfrq()) / 1000000ULL)
```

Comment:
Could be a static inline function instead

Reply:
```
static inline uint64_t us2cnt(uint32_t us)
{
	return ((uint64_t)us * (uint64_t)read_cntfrq()) / 1000000ULL;
}
```
Look strange a global function `us2cnt()`. Maybe `arm_cnt_us2cnt()`?